src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -166,35 +209,34 @@ const AssetContainer: React.FC<Props> = ({
 };
 
 const Wrapper = styled.div`
-  height: 558px;
   width: 100%;
 `;
 
-const AssetWrapper = styled(Flex)`
-  height: 425px;
+const AssetWrapper = styled.div<{ isHeightFixed?: boolean }>`
+  height: ${({ isHeightFixed }) => (isHeightFixed ? "" : "425px")};
+  min-height: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const ButtonWrapper = styled.div<{ onRemove?: (assets: AssetType[]) => void }>`
```

Comment:
Any reason this isn't a Flex component? If it is a Flex component you don't need to make a styled component here. Then just do something like:
`<Flex justify={onRemove ? "flex......>....</Flex>`

Reply:
ok, I wil fix it